### PR TITLE
fix: broken tests

### PR DESCRIPTION
**Pull request by duanyutong**
fix: broken tests

### CODE REVIEW

File: tests/dracula/test_run_topp.py [hunk 1]
```diff
@@ -74,14 +96,11 @@ def test_run_topp_spline_random_data():
...
-        vlim = np.asarray([1] * n_dof)
-        alim = np.asarray([2] * n_dof)
-        vlim = np.vstack([-vlim, vlim]).T
-        alim = np.vstack([-alim, alim]).T
         for n_waypts in [2, 20, 50, 200]:  # , 2000]:
             print(f"Testing {n_waypts} random waypoints...")
             waypts = np.random.rand(n_waypts, n_dof)
-            tdrac.run_topp_spline(waypts, vlim, alim, verify_lims=True)
+            qlim, vlim, alim = TestRunTopp._gen_limits(waypts)
+            tdrac.run_topp_spline(waypts, qlim, vlim, alim, verify_lims=True)
```
Comment (alanxzhou) [inline tests/dracula/test_run_topp.py:66]: looks like the `alim` and `vlim` have different values now than the old test. I'm guessing that's not a problem?

File: tests/dracula/test_run_topp.py [hunk 2]
```diff
@@ -28,6 +28,26 @@ def _gen_limits(waypts):
...
+            1.7628,
+            2.8973,
+            -0.0698,
+            2.8973,
+            3.7525,
+            2.8973,
+        ]
+        qlim[:-7, 0] *= -1
         v_max = 3 * np.ones((n_dof, 2))  # init with 3
         a_max = 3 * np.ones((n_dof, 2))  # init with 3
```
Comment (alanxzhou) [inline tests/dracula/test_run_topp.py]: Now that we're using these arrays directly as `vlim` and `alim` maybe rename them? Don't have strong opinions on this though

**Review by alanxzhou — APPROVED** [review]
lgtm. A couple of non-blocking comments/questions

**Comment by sadraddini** [comment]
Why do we even care about this repo anymore?

**Comment by duanyutong** [comment]
> Why do we even care about this repo anymore?

bad actions missing timing info has to be time-optimised on the fly in reduction

File: tests/dracula/test_run_topp.py (hunk 1 above)
Comment (duanyutong) [reply tests/dracula/test_run_topp.py:66]: I don't think any values changed. It's just the tests being outdated with wrong call signatures

File: tests/dracula/test_run_topp.py (hunk 2 above)
Comment (duanyutong) [reply tests/dracula/test_run_topp.py]: this is now a library function that provides easy access to the stock limits at their max

**Pull request merged**